sdk/keyvault/keyvault-admin/tests.yml

```diff
@@ -1,25 +1,23 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/keyvault-admin"
-    ResourceServiceDirectory: keyvault
-    # KV HSM limitation prevents us from running live tests
-    # against multiple platforms in parallel (we're limited to a single
-    # instance per region per subscription) so we're only running
-    # live tests against a single instance.
-    TestMinMax: false
-    TestBrowser: false
-    testSamples: false
-    ArmTemplateParameters: "@{ enableHsm = $true }"
-    Matrix:
-      Linux Node 10 with Managed HSM:
-        Pool: azsdk-pool-mms-ubuntu-1804-general
-        OSVmImage: MMSUbuntu18.04
-        TestType: "node"
-        NodeTestVersion: "10.x"
-    EnvVars:
-      AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-      AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/keyvault-admin"
+      ServiceDirectory: keyvault
+      # KV HSM limitation prevents us from running live tests
+      # against multiple platforms in parallel (we're limited to a single
+      # instance per region per subscription) so we're only running
+      # live tests against a single instance.
+      Location: eastus2
+      MatrixFilters:
+        - TestType=node
```

Comment:
Why do we need the MatrixFilter here? It looks like it should just be everything in this matrix.

Reply:
This replaces:

```
TestSamples: false
TestBrowser: false
```

I figured the inclusive filter was more readable here than the exclusive regex.

Reply:
Oh, this one has a custom matrix, yeah I can delete this.